src/os-utils/is-windows.spec.js

```diff
@@ -0,0 +1,15 @@
+import osTypes from '../constants/os-types'
+import isWindows from './is-windows';
+
+const getParserMock = (osName) => ({
+  getOSName: jest.fn(() => osName)
+});
+
+describe('OS utils: ', () => {
```

Comment:
Each spec should have the following structure (refs: https://github.com/yola/wiki/wiki/Javascript-Testing)

```js
describe('isWindows', () => {
  describe('when OS is Windows', () => {
    it('should return true', () => {
```